pkg/kubelet/kubelet.go

```diff
@@ -1900,7 +1919,11 @@ func (kl *Kubelet) syncPod(o syncPodOptions) error {
 	}
 
 	// Wait for volumes to attach/mount
-	if err := kl.volumeManager.WaitForAttachAndMount(pod); err != nil {
+	defaultedPod, _, err := kl.defaultPodLimitsForDownwardApi(pod, nil)
```

Comment:
It was called in getNode and getNodeAnyway

I don't like initial node status modifying Kubelet state.  I will move setting last node update object into Kubelet Run when setup to not register node based on the value returned from initial node status.

This change will have to wait until tomorrow.